pyswmm/simulation.py

```diff
@@ -17,7 +17,7 @@
 
 def output_exception_handler(func):
```

Comment:
Sometimes we just want to open an output of an already completed simulation and not one where we have executed in the current sessions. Would we have to do this outside of pyswmm?

Reply:
I agree it is often critical to be able to read past .out files - if this cannot be done in PySWMM it would be a welcome addition.

Reply:
Yes. The user can create an output object instead of using sim.

I was thinking something like this:
```
with Output(binfile_path) as out:
    out.output_function()
```

Reply:
I will add an example in unit test later

Reply:
@cbuahin Here is an example of how output file will function: 
https://github.com/jennwuu/pyswmm/blob/de66652d69c7821f062c1e8dc1b929921c77a4fb/pyswmm/tests/test_output.py#L6-L25

Reply:
It's not possible to connection Simulation object with Output object. So I am keeping them separate for now.